contracts/tokens/SecurityToken.sol

```diff
@@ -102,14 +102,6 @@ contract SecurityToken is ERC20, ERC20Detailed, ReentrancyGuard, RegistryUpdater
     event UpdateTokenDetails(string _oldDetails, string _newDetails);
     // Emit when the granularity get changed
     event GranularityChanged(uint256 _oldGranularity, uint256 _newGranularity);
-    // Emit when Module get archived from the securityToken
```

Comment:
This will remove the existence of these events from the SecurityToken ABI. So these events may not reflect in the etherscan until unless etherscan handling the events emitted by the liniking library. I am not sure but need to test this behavior.

Reply:
OK - thanks - will add a note to test this.